### DISCUSSION TITLE
Jitsy...

### DISCUSSION THREAD

**Discussion by 111phil** [OpenSlides/OpenSlides#6365] [General]
Hi! As probably many of you have already experienced, the Jitsy video conference tool is unfortunately quite disappointing.

1. Many of our participants have trouble using it, e.g. because the browser demands mic and cam access, which only pops up once and then doesn't show up again.
2. Moreover, if you have different tabs open, it opens new videos for all of them, which is not only annoying, but also waisting a lot of bandwith.
3. Lastly, it could be helpful, if there was the possibility of showing different videos at the same time (e.g. for deaf sign interpreters).

**Comment by ostcar** [comment]
I created this issue in the client repo: https://github.com/OpenSlides/openslides-client/issues/857

**Comment by jsangmeister** [comment]
> 2. Moreover, if you have different tabs open, it opens new videos for all of them, which is not only annoying, but also waisting a lot of bandwith.

What are you referring to exactly? You can pause the livestream and hide it, so it doesn't bother you anymore. Yu have to enter the jitsi conference manually for each tab, so this should not be a problem, either.

**Comment by tsiegleauq** [comment]
> Moreover, if you have different tabs open, it opens new videos for all of them, which is not only annoying, but also waisting a lot of bandwith.

I remember that I addressed this behavior already, so that you would get a "jitsi is already active in another tab" information. This should still be present in both OS3 and OS4.

Manually stopping the live stream (which is district from jitsi) was not desired
